exercises/armstrong-numbers/canonical-data.json

```diff
@@ -1,7 +1,15 @@
 {
   "exercise": "armstrong-numbers",
-  "version": "1.0.0",
+  "version": "2.0.0",
```

Comment:
This should be `1.1.0`

Reply:
Indeed! I misread [_"Major version changes"_](https://github.com/exercism/problem-specifications#major-version-changes) assuming "new property" was a new test case and skimmed over "Minor version changes" where this was made clear. Thanks!